.pipelines/build-pipeline.yml

```diff
@@ -46,17 +46,34 @@ steps:
     workingDirectory: src
 
 - task: CopyFiles@2
-  displayName: Copy artifacts
+  displayName: Copy packaging artifacts
   inputs:
     contents: |
       src/**/*.nupkg
       src/**/*.snupkg
+      src/Dto/EmsApi.Dto.V2.cs
+      ReleaseNotes.md
+    targetFolder: $(Build.ArtifactStagingDirectory)/packages
+    flattenFolders: true
+
+- task: CopyFiles@2
+  displayName: Copy netstandard2.0 artifacts
+  inputs:
+    contents: |
       src/Dto/bin/$(buildConfiguration)/netstandard2.0/EmsApi.Dto.dll
       src/Dto/bin/$(buildConfiguration)/netstandard2.0/EmsApi.Dto.pdb
       src/Client/bin/$(buildConfiguration)/netstandard2.0/EmsApi.Client.dll
       src/Client/bin/$(buildConfiguration)/netstandard2.0/EmsApi.Client.pdb
-      src/Dto/EmsAi.Dto.V2.cs
-    targetFolder: $(Build.ArtifactStagingDirectory)
+    targetFolder: $(Build.ArtifactStagingDirectory)/netstandard2.0
+    flattenFolders: true
+
+- task: CopyFiles@2
+  displayName: Copy netstandard2.1 artifacts
+  inputs:
+    contents: |
+      src/Client/bin/$(buildConfiguration)/netstandard2.1/EmsApi.Client.dll
```

Comment:
Do we not want the DTO dll and pdb for 2.1 as well?

Reply:
I had wanted to do this but ran into some issues with the generated DTO file hitting some access conflicts when applying the post-build replacements we do with msbuild. I guess msbuild is doing some kind of parallelism when you build for multiple targets. Anyway I might poke around at it some more today.

Reply:
Okay got this fixed up with some wacky msbuild hijinks